### PR TITLE
Updated mmctl export create command flag

### PR DESCRIPTION
**Pull request by cwarnermm**
#### Summary
Updated mmctl export create command flag to align with product documentation (updated via https://github.com/mattermost/docs/pull/6796)

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by hanzei — CHANGES_REQUESTED** [review]
You need to update the source go file. The rst docs are auto-generated.

**Comment by cwarnermm** [comment]
Oh wow! Thanks, @hanzei! It's been so long since I made changes here, I'd completely forgotten about that.

File: server/cmd/mmctl/docs/mmctl_export_create.rst [hunk 1]
```diff
@@ -21,8 +21,8 @@ Options
 ::
 
   -h, --help                        help for create
-      --include-archived-channels   Set to true to include archived channels in the export file.
-      --no-attachments              Set to true to exclude file attachments in the export file.
+      --include-archived-channels   Include archived channels in the export file.
+      --no-attachments              Omit file attachments in the export file.
```
Comment (cwarnermm) [inline server/cmd/mmctl/docs/mmctl_export_create.rst:25]:
```suggestion
      --no-attachments              Exclude file attachments from the export file.
```

**Comment by cwarnermm** [comment]
@hanzei - Given that the mmctl docs are auto-generated, is it necessary for me to revert the doc changes, or will they simply be overwritten with the Go changes?

**Review by JulienTant — APPROVED** [review]

**Comment by JulienTant** [comment]
@cwarnermm I believe the process is to change the go file, then run `make mmctl-docs`, and commit the result. Chances are that your manual changes will match exactly what the tool would have done automatically and you'll have nothing to commit! :)

**Comment by cwarnermm** [comment]
@JulienTant - When I ran the ``make mmctl-docs`` command within my local branch, I see the error: ``make: *** No rule to make target `mmctl-docs'.  Stop.``. Not sure how to proceed. Any suggestions?

**Comment by hanzei** [comment]
@cwarnermm Did you run the command from within `server`?

**Comment by cwarnermm** [comment]
Thanks, @hanzei! I re-ran the command via the ``server`` directory. This time the result was a different error: Error 1.

```
M1 detected, applying elasticsearch override
rm -rf ./cmd/mmctl/docs
cd ./cmd/mmctl && go run mmctl.go docs
# github.com/mattermost/mattermost/server/v8/einterfaces
../../einterfaces/cloud.go:57:46: undefined: model.AllowedIPRanges
../../einterfaces/cloud.go:58:38: undefined: model.AllowedIPRanges
../../einterfaces/ip_filtering.go:9:40: undefined: model.AllowedIPRanges
../../einterfaces/ip_filtering.go:10:25: undefined: model.AllowedIPRanges
# github.com/mattermost/mattermost/server/v8/channels/store
../../channels/store/store.go:124:70: undefined: model.RetentionIdsForDeletion
../../channels/store/store.go:740:35: undefined: model.RetentionIdsForDeletion
make: *** [mmctl-docs] Error 1
```

**Comment by cwarnermm** [comment]
Unfortunately, that too wasn't successful, @JulienTant.

**Review by hanzei — APPROVED** [review]
LGTM :+1: 

Thanks for updating _all_ the sources. Looking forward to simplifying the process.

**Pull request merged**